conformance/utils/roundtripper/roundtripper.go

```diff
@@ -182,7 +181,7 @@ func (d *DefaultRoundTripper) CaptureRoundTrip(request Request) (*CapturedReques
 		fmt.Printf("Received Response:\n%s\n\n", formatDump(dump, "< "))
 	}
 
-	body, _ := iou.ReadAll(resp.Body)
+	body, _ := io.ReadAll(resp.Body)
```

Comment:
Side comment, why not check error here?

Reply:
I have the same doubts. Let's see what other people think, should I add error checking?